src/commands/Admin/admin/botfarm.ts

```diff
@@ -0,0 +1,71 @@
+import Command from "../../../structures/Command";
+import SubCommand from "../../../structures/SubCommand";
+import CommandContext from "../../../structures/CommandContext";
+import { Guild, Message } from "discord.js";
+import BulbBotClient from "../../../structures/BulbBotClient";
+import { writeFileSync } from "fs";
+
+export default class extends SubCommand {
+	constructor(client: BulbBotClient, parent: Command) {
+		super(client, parent, {
+			name: "botfarm",
+			minArgs: 1,
+			maxArgs: 1,
+			argList: ["percent:number"],
+			usage: "<percent>",
+		});
+	}
+
+	public async run(context: CommandContext, args: string[]): Promise<void | Message> {
+		this.client.guilds.cache.map(async (guild: Guild) => await guild.members.fetch());
+
+		let botfarms: any = [];
+		let isFile: boolean = false;
+
+		const whitelisted: string[] = ["742094927403679816", "784408056997216327", "818176562901549066", "820945336327602186"];
+
+		this.client.guilds.cache.map(g => {
+			if (Math.round((g.members.cache.filter(u => u.user.bot).size / g.members.cache.size) * 100) < parseInt(args[0])) return;
+			if (whitelisted.includes(g.id)) return;
+
+			botfarms.push({
+				id: g.id,
+				name: g.name,
+				members: g.members.cache.filter(u => !u.user.bot).size,
+				bots: g.members.cache.filter(u => u.user.bot).size,
+				botP: Math.round((g.members.cache.filter(u => u.user.bot).size / g.members.cache.size) * 100),
+			});
+		});
+
+		botfarms = await botfarms.sort(dynamicSort("botP"));
```

Comment:
```suggestion
		botfarms.sort(dynamicSort("botP"));
```
[Array#sort](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort) is an in-place sorting method, so `botfarms` is already updated. `await` is also not needed.